### ISSUE TITLE
anti-LPL autoantibody

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#10511]
**Preferred term label:**
anti-lipoprotein lipase autoantibody positivity

**Synonyms**
anti-LPL autoantibody positivity

**Definition (free text, please give PubMed ID)**
PMID: 29784574|15840743|36123759|28916403

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:444490

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
@MaudDLVT 
Adding new term: 
- ID: HP:6001060
- label: Anti-lipoprotein lipase autoantibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:29784574|PMID:15840743|PMID:36123759|PMID:28916403
- def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against anti-lipoprotein lipase.
- comment: 
- orcid: orcid:0000-0002-3387-1836
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10511
- synonyms: Anti-LPL autoantibody positivity(NONE)
